x-pack/test_serverless/api_integration/test_suites/common/reporting/datastream.ts

```diff
@@ -23,35 +27,43 @@ export default function ({ getService }: FtrProviderContext) {
 
   describe('Data Stream', () => {
     before(async () => {
+      role = await svlUserManager.createApiKeyForRole('admin');
```

Comment:
In other tests, instead of `role` you used `roleAuthc` which I think describes it better. Was there a reason to name it differently here?

Reply:
https://github.com/elastic/kibana/pull/184819/commits/96d703359ed2ea793f8c8c0d47499e535a157190